### ISSUE TITLE
The links from Eupnea Project have been deleted!

### ISSUE THREAD

**Issue by hamed-ta** [olm3ca/Pixel-Slate#1]
The links from Eupnea Project in the linux audio part are deleted.

**Comment by olm3ca** [comment]
You're right - you can find them now here: https://github.com/WeirdTreeThing/chromebook-linux-audio

I've updated the instructions. Thanks
